pkg/yaml/yaml_test.go

```diff
@@ -52,29 +48,17 @@ func TestReadYamlAllFileFs(t *testing.T) {
 	// Setup temporary file
 	file, err := CreateTempFile(t, existingEmptyYamlFileNamePattern)
 	defer file.Close()
-	if err != nil {
-		t.Errorf("Can't create file: %s", file.Name())
-	}
+	assert.NoError(t, err, "Can't create file: %s", file.Name())
 
 	//Read existing empty yaml file
 	existingEmptyYamlAllFileResult, existingEmptyYamlAllFileErr := ReadYamlAllFile(file.Name())
-	if existingEmptyYamlAllFileErr != nil {
-		t.Errorf("Can't read empty yaml file: %s", file.Name())
-	}
-
-	if existingEmptyYamlAllFileResult != nil {
-		t.Errorf("Empty YAML stream read incorrectly. Value should be nil")
-	}
+	assert.NoError(t, existingEmptyYamlAllFileErr, "Can't read empty yaml file: %s", file.Name())
+	assert.Nil(t, existingEmptyYamlAllFileResult, "Empty YAML stream read incorrectly. Value should be nil")
```

Comment:
assert.Empty() is the better fit here, as nil and an empty slice are logically the same